testrpc/client/client.py

```diff
@@ -49,7 +50,6 @@
 DEFAULT_GAS_LIMIT = t.gas_limit = t.GAS_LIMIT = int(os.environ.get('TESTRPC_GAS_LIMIT', 4000000))
 config.default_config['GENESIS_GAS_LIMIT'] = DEFAULT_GAS_LIMIT
 
-
```

Comment:
This whitespace was here to visually separate these two lines.  Please re-add it.